CIP-conway-keys/README.md

```diff
@@ -0,0 +1,245 @@
+Such keys are to be known as DRep keys, constitutional committee cold keys and constitutional committee hot keys.
+Here we define some accompanying tooling standards.
+
+> **Note** this proposal assumes knowledge of the Conway ledger design (see
+> [draft ledger specification](https://github.com/input-output-hk/cardano-ledger/blob/d2d37f706b93ae9c63bff0ff3825d349d0bd15df/eras/conway/impl/cddl-files/conway.cddl))
+> and
+> [CIP-1694](https://github.com/cardano-foundation/CIPs/blob/master/CIP-1694/README.md).
+
+## Motivation: why is this CIP necessary?
+
+In the Conway ledger era, DRep credentials allow registered DReps to be identified on-chain, in DRep registrations, retirements, votes, and in vote delegations from ada holders.
+Whilst constitutional committee members can be recognized by their cold credentials within update committee governance actions, authorize hot credential certificate and resign cold key certificates.
+Constitutional committee hot credential can be observed within the authorize hot key certificate and votes.
+
+CIP-1694 terms these DRep credentials as DRep IDs, which are either generated from blake2b-224 hash digests of Ed25519 public keys owned by the DRep, or are script hash-based.
+Similarly, both the hot and cold credentials for constitutional committee members can be generated from public key digests or script hashes.
+
+This CIP defines a standard way for wallets to derive DRep and constitutional committee keys.
+
+Since it is best practice to use a single cryptographic key for a single purpose, we opt to keep DRep and committee keys separate from other keys in Cardano.
+
+By adding three paths to the [CIP-1852 | HD (Hierarchy for Deterministic) Wallets for Cardano](https://github.com/cardano-foundation/CIPs/blob/master/CIP-1852/README.md), we create an ecosystem standard for wallets to be able to derive DRep and constitutional committee keys.
+This enables DRep and constitutional committee credential restorability from a wallet seed phrase.
+
+Stakeholders for this proposal are wallets that follow the CIP-1852 standard and tool makers wishing to support DReps and or constitutional committee members.
+This standard allows DReps and constitutional committee members to use alternative wallets whilst being able to be correctly identified.
+By defining tooling standards, we enable greater interoperability between governance-focussed tools.
+
+## Specification
+
+### Derivation
+
+#### DRep Keys
+
+Here we describe DRep key derivation as it pertains to Cardano wallets that follow the CIP-1852 standard.
+
+To differentiate DRep keys from other Cardano keys, we define a new `role` index of `3`:
+
+`m / 1852' / 1815' / account' / 3 / address_index`
+
+We strongly recommend that a maximum of one set of DRep keys should be associated with one wallet account, which can be achieved by setting `address_index=0`.
+
+#### DRep ID
```

Comment:
Out of curiosity, why is this called an _ID_ in the title rather than a _Credential_ as in the 2 headings below for the other keys?

Reply:
I suspect because that is what this is called in the spec. Probably also because it is  used for uniquely identifying the DRep.

We similarly call Pool's KeyHash as PoolId in Ledger.

Reply:
I got it from CIP-1694, which is very close to the ledger spec :)